### ISSUE TITLE
Manage Subtitles

### ISSUE THREAD

**Issue by lowlevel-1989** [danamag/stremio-addons-list#211]
### Addon Manifest URL

https://addons.lowlevel1989.click/conf/api/v1/manifest.json

### Addon Description

**Manage Subtitles** is an add-on for Stremio that allows you to manage your subtitles or follow the subtitles of other users completely free of charge.

### Language of Content

None

### Choose Labels

- [ ] anime
- [ ] asian drama
- [ ] bollywood
- [ ] cartoons
- [ ] debrid support
- [ ] http streams
- [ ] live tv
- [ ] metadata
- [ ] misc
- [ ] movies
- [ ] music
- [ ] nsfw
- [ ] podcasts
- [ ] radios
- [X] subtitles
- [ ] torrents
- [ ] tv shows
- [ ] video games

**Comment by kaissersuze** [comment]
Excellent!!!

**Comment by SPICEG0D** [comment]
how exactly do we add/configure this one? thanks in advance :D

**Comment by lowlevel-1989** [comment]
> how exactly do we add/configure this one? thanks in advance :D

Usage of 'Manage Subtitles' varies depending on your specific needs:

- If you only wish to utilize the service's subtitles, simply configure it in Stremio through the install button. We recommend using the registered version for a more comprehensive experience.

- For more personalized subtitles, it's best to access the dashboard and follow subtitles that interest you, contributed by other users with public profiles.

- If you want to upload your own subtitles, you'll need to create a repository on GitHub or any other platform of your choice and then upload them to the 'Manage Subtitles' service, following the detailed instructions provided in the manual, available here: [Link to the manual](https://addons.lowlevel1989.click/dashboard/manual/).

**Comment by lowlevel-1989** [comment]
> how exactly do we add/configure this one? thanks in advance :D

Starting from version 0.23.0.1, we have resolved an issue with the 'Manage Subtitles' install button that previously made it impossible to install the add-on without using the website.

We apologize for any inconvenience this may have caused, but we are pleased to announce that the new implementation is now operational. Thank you for your understanding.

-------

A partir de la versión 0.23.0.1, hemos corregido un error que impedía la instalación del complemento 'Manage Subtitles' a través del botón de instalación, obligando a los usuarios a recurrir al sitio web para la instalación.

Lamentamos las molestias ocasionadas, pero nos complace informar que la nueva implementación ya está en funcionamiento. Agradecemos su comprensión y paciencia.

**Comment by AIWintermuteAI** [comment]
Hi!
Is there any instruction on how to get it to work in Stremio? I installed the addon, but I'm not seeing any interface element to choose subtitles from in Stremio, except for normal OpenSubtitles add-on.

**Comment by lowlevel-1989** [comment]
> Hi! Is there any instruction on how to get it to work in Stremio? I installed the addon, but I'm not seeing any interface element to choose subtitles from in Stremio, except for normal OpenSubtitles add-on.

Hello, sure, to add subtitles in Stremio, you simply need to install the "Manage Subtitles" addon. If you already have it installed, the subtitle you want should be available on the platform ([dashboard](https://addons.lowlevel1989.click/dashboard/subtitle/)), meaning that another user has uploaded it following the instructions in the dashboard [manual](https://addons.lowlevel1989.click/dashboard/manual/). If not, you will need to refer to the manual to learn how to upload the subtitle to the platform, allowing both you and other users to use it.

What subtitle do you need and in which language? If I have some time today, I'll be happy to upload it for you. However, please keep in mind that it usually takes me more time to upload them as there are many pending requests.

I hope this information is helpful. Best regards!

**Comment by napster2142** [comment]
> > Hi! Is there any instruction on how to get it to work in Stremio? I installed the addon, but I'm not seeing any interface element to choose subtitles from in Stremio, except for normal OpenSubtitles add-on.
> 
> Hello, sure, to add subtitles in Stremio, you simply need to install the "Manage Subtitles" addon. If you already have it installed, the subtitle you want should be available on the platform ([dashboard](https://addons.lowlevel1989.click/dashboard/subtitle/)), meaning that another user has uploaded it following the instructions in the dashboard [manual](https://addons.lowlevel1989.click/dashboard/manual/). If not, you will need to refer to the manual to learn how to upload the subtitle to the platform, allowing both you and other users to use it.
> 
> What subtitle do you need and in which language? If I have some time today, I'll be happy to upload it for you. However, please keep in mind that it usually takes me more time to upload them as there are many pending requests.
> 
> I hope this information is helpful. Best regards!

Hi, greetings

I have follow everything in the manual given on your web, but when i click upload subtitle, nothing happen, it reset back to blank, is the web down or I did something wrong somewhere?

Thanks

**Comment by lowlevel-1989** [comment]
> > > Hi! Is there any instruction on how to get it to work in Stremio? I installed the addon, but I'm not seeing any interface element to choose subtitles from in Stremio, except for normal OpenSubtitles add-on.
> > 
> > 
> > Hello, sure, to add subtitles in Stremio, you simply need to install the "Manage Subtitles" addon. If you already have it installed, the subtitle you want should be available on the platform ([dashboard](https://addons.lowlevel1989.click/dashboard/subtitle/)), meaning that another user has uploaded it following the instructions in the dashboard [manual](https://addons.lowlevel1989.click/dashboard/manual/). If not, you will need to refer to the manual to learn how to upload the subtitle to the platform, allowing both you and other users to use it.
> > What subtitle do you need and in which language? If I have some time today, I'll be happy to upload it for you. However, please keep in mind that it usually takes me more time to upload them as there are many pending requests.
> > I hope this information is helpful. Best regards!
> 
> Hi, greetings
> 
> I have follow everything in the manual given on your web, but when i click upload subtitle, nothing happen, it reset back to blank, is the web down or I did something wrong somewhere?
> 
> Thanks

Yes, it was recently undergoing maintenance. Could you please try again and let me know if you encounter any other issues? If so, a screenshot would be helpful.

Best regards,

**Comment by napster2142** [comment]
Thanks @lowlevel-1989 , I have succesfully uploaded the subtitle, is there anyway to delete it? as i still searching for the perfect sync subtitle, so will need to have a trial and error process, worried that it will flood your library with the same movie subtitle

**Comment by Ilikecofe** [comment]
Hi, I'm having trouble trying to upload a raw YAML file. It keeps showing me this error: "Url: We exclusively support the SRT format, but for loading multiple items in the background, we also offer support for YAML."
I'm attaching the raw yaml url from github and getting that error. Also, I'm not sure what "season" I should put in the YAML code (it's a series with a Kitsu ID).
And when uploading, what should I put in "episode"? Should I just leave it blank?

Thanks in advance.

**Comment by lowlevel-1989** [comment]
> Hi, I'm having trouble trying to upload a raw YAML file. It keeps showing me this error: "Url: We exclusively support the SRT format, but for loading multiple items in the background, we also offer support for YAML." I'm attaching the raw yaml url from github and getting that error. Also, I'm not sure what "season" I should put in the YAML code (it's a series with a Kitsu ID). And when uploading, what should I put in "episode"? Should I just leave it blank?
> 
> Thanks in advance.

Muchas gracias, segun veo hablas español, es un bugs ya estamos solucionando el problema, tu yaml es correcto y puedes reportar el bugs en este linik por favor, para poder informarte cuando este solucionado y si es posible dime tu nombre de usuario.

**Comment by lowlevel-1989** [comment]
> Hi, I'm having trouble trying to upload a raw YAML file. It keeps showing me this error: "Url: We exclusively support the SRT format, but for loading multiple items in the background, we also offer support for YAML." I'm attaching the raw yaml url from github and getting that error. Also, I'm not sure what "season" I should put in the YAML code (it's a series with a Kitsu ID). And when uploading, what should I put in "episode"? Should I just leave it blank?
> 
> Thanks in advance.

fix this, please run the tests with your yaml: [https://raw.githubusercontent.com/Ilikecofe/letstrymsub/main/ani/snk/0401/SNKFSpart1.yaml](https://raw.githubusercontent.com/Ilikecofe/letstrymsub/main/ani/snk/0401/SNKFSpart1.yaml)

**Comment by Ilikecofe** [comment]
> > Hi, I'm having trouble trying to upload a raw YAML file. It keeps showing me this error: "Url: We exclusively support the SRT format, but for loading multiple items in the background, we also offer support for YAML." I'm attaching the raw yaml url from github and getting that error. Also, I'm not sure what "season" I should put in the YAML code (it's a series with a Kitsu ID). And when uploading, what should I put in "episode"? Should I just leave it blank?
> > Thanks in advance.
> 
> fix this, please run the tests with your yaml: https://raw.githubusercontent.com/Ilikecofe/letstrymsub/main/ani/snk/0401/SNKFSpart1.yaml

Hi, I'm still having issues uploading the yaml link. I just posted on the issues tab on github as suggested so we can continue there. Thanks
